drivers/sensor/lis2dh/lis2dh_trigger.c

```diff
@@ -407,56 +401,54 @@ int lis2dh_init_interrupt(const struct device *dev)
 #endif
 
 	/* data ready int1 gpio configuration */
-	status = gpio_pin_configure(lis2dh->gpio_int1, cfg->irq1_pin,
-				    GPIO_INPUT | cfg->irq1_flags);
+	status = gpio_pin_configure_dt(&cfg->gpio_drdy, GPIO_INPUT);
 	if (status < 0) {
-		LOG_ERR("Could not configure gpio %d", cfg->irq1_pin);
+		LOG_ERR("Could not configure gpio %d", cfg->gpio_drdy.pin);
```

Comment:
Error logging is dropping the device name here, which will make debugging harder if two ports have the same pin number and are used for DRDY. Not caused by this PR though. Note that the LOG_INF() at line 420 gets this right.